package.json

```diff
@@ -32,7 +32,12 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@ng-bootstrap/ng-bootstrap": "^10.0.0",
     "@tiptap/core": "^2.0.0-beta.204",
+    "@tiptap/extension-image": "^2.0.0-beta.209",
```

Comment:
This version should match the version of the other extensions, if possible. (`2.0.0-beta.204`)